src/components/controls/dropdown/Dropdown.js

```diff
@@ -69,7 +70,7 @@ function Dropdown({
 }
 
 Dropdown.propTypes = {
-  labelText: PropTypes.string.isRequired,
```

Comment:
Why was this changed to no longer be required, out of curiousity?

Reply:
The simple reason is because I wanted to make use of the dropdown we already created, but my use case doesn't have a label, and a label would be detrimental to the experience of the component. I removed it because it functions without one, and really should allow the user to decide if a label is necessary. If it is better I can put in a boolean that asks whether or not the label should be displayed so we don't get extra elements.

Reply:
Rather than a boolean, you could just check for the existence of the labelText prop and conditionally render the label elements.

Reply:
sounds good to me, I will get on that.